cfdm/mixin/fielddomain.py

```diff
@@ -1853,7 +1853,12 @@ def equals(
 
         """
         # Check the properties and data
-        ignore_properties = tuple(ignore_properties) + ("Conventions",)
+        if not ignore_properties:
+            ignore_properties = ("Conventions",)
```

Comment:
Since the "Conventions" property is always ignored, as clearly stated in the docstring parameter description, is it slightly misleading to have a default of `None` for the keyword when we always ignore that anyway? Could/should we change the default to `("Conventions",)`? (Just throwing that question out there, not sure where I sit myself.)

Reply:
Hi - not sure. If the default was `("Conventions",)` then it would imply that you can remove it, which you can't, and you would have to add it in manually if you set your own property.

Reply:
Oh, so if you can't remove it (context I didn't know though I probably should know, sorry :grimacing:) then I agree we should leave the default and set up as it is. Marking as resolved!